packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.component.js

```diff
@@ -102,16 +102,11 @@ export class CheckoutOrderSummary extends PureComponent {
             return null;
         }
 
-        if (!coupon_code) {
-            return this.renderPriceLine(
-                -Math.abs(discount_amount),
-                __('Discount %s:', '')
-            );
-        }
+        const label = coupon_code ? __('Discount/Coupon %s:', coupon_code.toUpperCase()) : __('Discount %s:', '');
```

Comment:
Very strange line, I do not understand what it does: `__('Discount %s:', '')`, hmm!